lib/documents.js

```diff
@@ -106,8 +108,9 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
  * @returns {Promise<object>} An object with the registration record.
  */
 export async function register({
-  internalId, externalId, document, recipients, recipientChain, ttl, creator,
-  tokenizer, externalIdHash, documentHash, creatorHash, newRegistration
+  internalId, externalId, document, recipients, recipientChain,
+  minAssuranceForResolution, ttl, creator, tokenizer, externalIdHash,
```

Comment:
Is the intent for this to be a breaking change? If not, we need a default. Reading back over the design doc indicates we should pick `2` as the default. We'll want to be careful how we set this default, however, only doing so when the registration is new and not a refresh of an existing one. In other words, the presence of `minAssuranceForResolution` indicates that we want to set its value to what was passed vs. taking one of two paths:

1. When registering a new one, use the value passed OR the default if one wasn't passed.
2. When refreshing an existing registration, use the value passed OR do NOT update `minAssuranceForResolution`.

This brings me to the fact that the `_refresh` path also needs consideration here to allow us to update `minAssuranceForResolution` on an already registered document.

That refresh is currently quite fast -- which we don't want to change. However, since we are storing `minAssuranceForResolution` in two places now -- both in token batches and in registered documents, we need to figure out how to keep them in sync or decide that we don't want to/need to.

I'm going to take this off to a design doc to discuss what we want to do.